lib/tesla_api/stream.rb

```diff
@@ -1,8 +1,9 @@
 module TeslaApi
   module Stream
-    def stream(&receiver)
+    def stream(endpoint: nil, &receiver)
+      endpoint ||= streaming_endpoint
```

Comment:
This can just be the default value instead.

```suggestion
    def stream(endpoint: streaming_endpoint, &receiver)
```

Reply:
Yeah – at the time there was a reason I didn't do it that way, but I can't think of it right now...

Likely, it was late and my mind wasn't on straight.  In any case, changes made.